assets/src/components/reporting/ReportingDashboard.tsx

```diff
@@ -171,6 +183,16 @@ class ReportingDashboard extends React.Component<Props, State> {
     });
   };
 
+  formatResponseTimeByWeekDay = (stats: Array<ResponseTimeByWeekDay>) => {
+    return stats.map((average: string, day) => {
```

Comment:
@cheeseblubber this breaks the frontend build -- should be `({average, day})` in curly brackets

(if you're using vscode, it might be helpful to install the `TSLint` plugin and the `Prettier - Code formatter` plugin to help catch things like this)